plugins/scaffolder/api-report.md

```diff
@@ -80,6 +80,20 @@ export const EntityPicker: ({
 // @public (undocumented)
 export const EntityPickerFieldExtension: () => null;
 
+// Warning: (ae-missing-release-tag) "EntityTagsPicker" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
I know there's a lot of existing warnings, but would be nice to not add more x)

Reply:
updated!